src/dbCache.js

```diff
@@ -0,0 +1,50 @@
+const insertImage = 'INSERT INTO images (id, x, y, fit, file_type, url, blur) VALUES ($1,$2,$3,$4,$5,$6,$7)';
+const selectImage = 'SELECT url FROM images WHERE id=$1 AND x=$2 AND y=$3 AND fit=$4 AND file_type=$5 AND blur=$6';
+
+export default (db) => {
+  const promiseQuery = (query, vars) => {
+    return new Promise((resolve, reject) => db.query(query, vars, (err, data) => err ? reject(err) : resolve(data)));
+  };
+  return {
+    getFromCache: async(params) => {
```

Comment:
Pro-typ: `async` also works well together with the method syntax: `async getFromCache(params) {}`.